### PR TITLE
 Update `Checks` to use current implementation

### PR DESCRIPTION
**Pull request by wojake**
Create, cancel, sign, submit, and query Checks are using depreciated RippleAPI. Update them to use current implementation.

This is work for a XRPL Bounty @ [XRPLBounties/Proposals](https://github.com/XRPLBounties/Proposals/blob/main/open_bounties/0077%20JavaScript%20Code%20Samples%202.md) for Milestone 1

### CODE REVIEW

File: content/_code-samples/checks/js/create-check.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    console.log(`\n The sender may cancel the Check after ${xrpl.rippleTimeToISOTime(Expiration)}`)
+    
+    const EscrowCreate_tx_signed = wallet.sign(EscrowCreate_tx)
+
+    console.log("\n Transaction hash:", EscrowCreate_tx_signed.hash)
+  
+    const EscrowCreate_tx_result = await client.submitAndWait(EscrowCreate_tx_signed.tx_blob)
+    
+    console.log("\n Submit result:", EscrowCreate_tx_result.result.meta.TransactionResult)
+    console.log("    Tx content:", EscrowCreate_tx_result)
```
Comment (ckniffen) [inline content/_code-samples/checks/js/create-check.js]: Output checkId for use subsequent scripts.

Comment (wojake) [reply content/_code-samples/checks/js/create-check.js]: Made the changes, this is what you wanted right?

File: content/_code-samples/checks/js/cash-check.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            console.log(`\n${i+1}. Index (ObjectID/keylet): ${incoming_cashable[i].index}`)
+            console.log(` - Account: ${incoming_cashable[i].Account}`)
+            console.log(` - Destination: ${incoming_cashable[i].Destination}`)
+            console.log(` - Amount: ${incoming_cashable[i].SendMax} drops`)
+            console.log(` - PreviousTxnID: ${incoming_cashable[i].PreviousTxnID}`)
+        }
+
+        console.log(`\nWe're going to cash Check ${incoming_cashable[0].index} which was sent from ${incoming_cashable[0].Account} @ ${incoming_cashable[0].SendMax} drops...`)
+     
+        // Get the transaction's sequence, this transaction created this Escrow
```
Comment (JST5000) [inline content/_code-samples/checks/js/cash-check.js:66]: This comment isn't quite relevant for the Check snippet

File: content/_code-samples/checks/js/cash-check.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        const get_check_id = await client.request({
```
Comment (JST5000) [inline content/_code-samples/checks/js/cash-check.js:67]: This variable is never used

**Comment by JST5000** [comment]
Hey WoJake, we're planning on using Tushar's implementation for this currently. I will close this PR once that merge happens. (Delayed by fixing the link checker issue which is in both PRs - Wanted to keep you in the loop so you don't accidentally spend more time on it after our decision)

**Comment by wojake** [comment]
Seems like I've forgotten to update this PR (with your suggestions) 😅 . Thanks for the heads up!

**Pull request closed without merging**